clusterloader2/pkg/prometheus/manifests/exporters/kube-state-metrics/serviceMonitor.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app.kubernetes.io/name: kube-state-metrics
+    app.kubernetes.io/version: 2.0.0-rc.0
+  name: kube-state-metrics
+  namespace: kube-state-metrics-perf-test
+spec:
+  endpoints:
+  - honorLabels: true
+    interval: 30s
+    port: http-metrics
```

Comment:
On second thought I am not sure we want to ingest all the kube-state-metrics metrics it produces to the Prometheus instance during tests, as I do not think that will work with the current resources set for that Prometheus as it would produce a lot of series. 

I might drop this, we already check in our perf tests that the endpoint is not empty. @tariq1890 @brancz @mrueg thoughts for or against this? We would keep the telemetry endpoint that produces self metrics.

Reply:
I am okay with ingesting all metrics honestly. The perf test is also meant to be a sort of stress test, yes? If not, I am not too opposed to the limiting the exposed ksm metrics